cypress/support/commands.ts

```diff
@@ -0,0 +1,15 @@
+import '@testing-library/cypress/add-commands'
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      goTo(uri: string): void
+    }
+  }
+}
+
+Cypress.Commands.add('goTo', (uri: string) => {
```

Comment:
Instead of creating a new `goTo` command, I think you may be able to just set Cypress' `baseUrl` via an environment variable and then just `cy.visit('/examples/richtest')`, as per https://docs.cypress.io/guides/guides/environment-variables#Setting

Reply:
great suggestion thank you! Addressed